linux_os/guide/system/accounts/accounts-physical/screen_locking/smart_card_login/group.yml

```diff
@@ -11,3 +11,5 @@ description: |-
     In Red Hat Enterprise Linux servers and workstations, hardware token login
     {{% endif %}}
     is not enabled by default and must be enabled in the system settings.
+
+platform: machine
```

Comment:
NACK. If someone is able to login, that logon should be CAC/PIV enabled. Rules like this should not have ``platform: machine``

Reply:
It seems like an overkill to me, but if people wanna do it, we shall not prevent them.